dynawo/sources/Models/CPP/ModelNetwork/test/TestBus.cpp

```diff
@@ -618,6 +618,26 @@ TEST(ModelsModelNetwork, ModelNetworkBusDefineInstantiate) {
   ASSERT_NO_THROW(bus->setSubModelParameters(parametersModels));
   ASSERT_DOUBLE_EQUALS_DYNAWO(bus->getUMax(), 10.);
   ASSERT_DOUBLE_EQUALS_DYNAWO(bus->getUMin(), 1.);
+
+  // The tests on the parameter hasShortCircuitCapabilities are done using the model size.
+  shared_ptr<ModelBus> bus2 = createModelBus(false, false, false).first;
+  bus2->initSize();
+  ASSERT_DOUBLE_EQUALS_DYNAWO(bus2->sizeY(), 2.);
+  parametersModels.clear();
+  const std::string param3Name = "bus_hasShortCircuitCapabilities";
```

Comment:
We could also test  global parameter=true?

Reply:
Done